src/ServiceManagement/Services/Commands.Test/Websites/PublishAzureWebsiteProjectTests.cs

```diff
@@ -18,6 +17,57 @@ public class PublishAzureWebsiteProjectTests : WebsitesTestBase
     {
         [Fact]
         [Trait(Category.AcceptanceType, Category.CheckIn)]
+        public void PublishFromPackageWithoutSetParameters()
+        {
+            const string websiteName = "test-site";
+            const string package = "test-package";
+            string slot = null;
+            var connectionStrings = new Hashtable { ["DefaultConnection"] = "test-connection-string" };
```

Comment:
I've moved the hashtable initialization after the declaration to match the existing syntax in other test cases.